regexes/device/mobiles.yml

```diff
@@ -4537,11 +4572,13 @@ Sony:
 
 # Ergo
 Ergo:
-  regex: 'ERGO|(V550[_ ]Vision|B502 Basic|B504[_ ]Unit|A502[_ ]Aurum|A503[ _]Optima|Prime B400|A551 Sky 4G|A553 Power|A555 Universe|Force F500|GoTab GBT9|GoTab Gti8|V551[ _]Aura|V540_Level|V570_BIG_BEN|B506[_ ]INTRO)'
+  regex: 'ERGO|(V550[_ ]Vision|B502 Basic|B504[_ ]Unit|A502[_ ]Aurum|A503[ _]Optima|Prime B400|A551 Sky 4G|A553 Power|A555 Universe|Force F500|GoTab GBT9|GoTab Gti8|V551[ _]Aura|V540_Level|V570_BIG_BEN|B506[_ ]INTRO|A503-Optima)'
   device: 'smartphone'
   models:
     - regex: 'B502 Basic'
       model: 'B502 Basic'
+    - regex: 'A503-Optima'
+      model: 'B503 Optima'
```

Comment:
Is that correct?

Reply:
Yes, this is a mistake, I'm already fixing it